packages/ui/src/components/va-image/VaImage.demo.vue

```diff
@@ -1,217 +1,214 @@
 <template>
-  <VbDemo>
-    <div class="row">
-      <VbCard title="default">
-        <va-image
-          style="width: 200px; height: 200px;"
-          :src="getImagePath(200)"
-        />
-      </VbCard>
-      <VbCard title="Slots (loading, error)">
-        <button @click="newImage()">
+  <VbDemo class="flex flex-wrap space-x-4 space-y-4">
+    <VbCard title="Default">
+      <va-image
+        class="image"
+        :src="getImagePath(200)"
+      />
+    </VbCard>
+
+    <VbCard title="Progressive loading">
+      <va-image
+        class="image"
+        src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
```

Comment:
Thiking if we can have APi with smaller footprint (and current could stay for cases where you need more flexibility)?

I.e. something like 
```html
class="image"
src="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png"
src-placeholder="https://zos.alipayobjects.com/rmsportal/jkjgkEfvpUPVyRjUImniVslZfWPnJuuZ.png?x-oss-process=image/blur,r_50,s_50/quality,q_1/resize,m_mfit,h_200,w_200"
```

Reply:
Done.